### ISSUE TITLE
How to train the WideResNet38-FCN for Segmentation 

### ISSUE THREAD

**Issue by PkuRainBow** [mapillary/inplace_abn#93]
@pkontschieder @ducksoup @rotabulo 

Hi, I have tried to train the WideResNet38-FCN models but find that the performance is even worse than the ResNet-101-FCN model under the same training settings.

For example, on the validation of Cityscapes, their performance is 
_ResNet-101-FCN            75.90
WideResNet38-FCN       73.50_

Thus  I am wondering some details of your training:

**Q1**
Is the **WiderResNetA2** the model that you use?

**Q2**
If the answer to Q1 is yes, then I find that you use two dropout in your **WiderResNetA2**
```
                if mod_id == 4:
                    drop = partial(nn.Dropout2d, p=0.3)
                elif mod_id == 5:
                    drop = partial(nn.Dropout2d, p=0.5)
```
I am wondering do we need to remove these two dropout in our training on segmentation task.

**Q3**
In the forward stage of **WiderResNetA2**,  do you keep the _bn_out_ after the _self.mod7_?

Here is my modification of the related fragments,

```
    def forward(self, img):
        tuple_features = list()
        out = self.mod1(img)
        out = self.mod2(self.pool2(out))
        out = self.mod3(self.pool3(out))
        out = self.mod4(out)
        tuple_features.append(out)
        out = self.mod5(out)
        tuple_features.append(out)
        out = self.mod6(out)
        tuple_features.append(out)
        out = self.mod7(out)
        out = self.bn_out(out)
        tuple_features.append(out)
        return tuple_features
```

It would be great if you could help me to ensure that the performance of  **WiderResNetA2** is as good as expected.

**Comment by PkuRainBow** [comment]
@pkontschieder @ducksoup @rotabulo Could anyone share me advice?

**Comment by rotabulo** [comment]
@PkuRainBow the model we use is the one that you can obtain here: https://github.com/mapillary/inplace_abn/blob/master/test_vistas_single_gpu.py#L201 without modifications.

**Comment by AmeetR** [comment]
@rotabulo  do you have advice for training on other datasets?

**Comment by ducksoup** [comment]
In an effort to streamline the resolution process we are closing any issue not directly related with the InPlace-ABN _library and scripts_. Please use other channels if you need to contact us about our paper or the Mapillary Vistas dataset.
